Dockerfile

```diff
@@ -31,6 +31,7 @@ RUN rm -rf /github
 
 ## Create the gRPC client
 ENV import_style=commonjs
+ENV grpc_web_import_style=commonjs
```

Comment:
Can you revert that to the same name used by gRPC-web `grpc-web_out`?

Reply:
looks like docker not working with environment variables that contains dash. But it works perfectly when use underscore.

Reply:
looks like docker not working with environment variables that contains dash. But it works perfectly when use underscore.

Reply:
looks like docker not working with environment variables that contains dash. But it works perfectly when use underscore.